packages/foreman/foreman/foreman.spec

```diff
@@ -107,61 +108,15 @@ BuildRequires: rubygem(bigdecimal)
 BuildRequires: gettext
 BuildRequires: /usr/bin/ruby
 BuildRequires: rubygems
+BuildRequires: rubygem(bundler)
 BuildRequires: rubygem(rake) >= 0.8.3
 BuildRequires: rubygem(bundler_ext)
 BuildRequires: /usr/bin/npx
 BuildRequires: make
-BuildRequires: (rubygem(rss) or ruby-default-gems < 3.0)
 Requires: (rubygem(rss) or ruby-default-gems < 3.0)
 BuildRequires: (rubygem(rexml) or ruby-default-gems < 3.0)
 Requires: (rubygem(rexml) or ruby-default-gems < 3.0)
```

Comment:
Why do we need rexml, https://github.com/theforeman/foreman-packaging/pull/10393 does not say it :crying_cat_face:

Reply:
It also confused me